mailchimp.php

```diff
@@ -126,11 +126,44 @@ public static function supportedLanguages()
         return self::$supportedLanguages;
     }
 
+    /**
+     * @param Event $event
+     */
+    protected function checkTriggers($event)
```

Comment:
Could we call this something like `shouldSubscribe` and move the declaration below `handleSubscribe`?

- `Subscribe` is the operative verb here, so connecting the method names semantically makes sense to me.
- `handleSubscribe` is the *primary* action, with `checkTriggers`/`shouldSubscribe` being a subsidiary method. As such, I'd declare it below, but still adjacent to the main action.